doc/sources/guide/licensing.rst

```diff
@@ -0,0 +1,137 @@
+* `sdl2 <https://www.libsdl.org/license.php>`_
+* `glew <http://glew.sourceforge.net/glew.txt>`_
+* `gstreamer <https://github.com/GStreamer/gstreamer/blob/master/COPYING>`_
+  (if used)
+* image & audio libraries(e.g. |mixer|_)
+
+You'll probably need to check image and audio libraries manually (mostly
+begin with ``lib``). The ``LICENSE*`` files that belong to them should be
+included by PyInstaller, but are not by python-for-android and you need
+to find them.
+
+Windows (PyInstaller)
+---------------------
+
+.. |win32| replace:: pypiwin32
+.. _win32: https://pypi.python.org/pypi/pypiwin32
+
+To access Windows API, Kivy uses |win32|_ package. This package is released
+under `PSF license <https://opensource.org/licenses/Python-2.0>`_.
+
+VS redistributables
+~~~~~~~~~~~~~~~~~~~
+
+.. |py2crt| replace:: Py2 CRT license
+.. _py2crt: https://hg.python.org/sandbox/2.7/file/tip/Tools/msi/crtlicense.txt
+.. |py3crt| replace:: Py3 CRT license
+.. _py3crt: https://hg.python.org/cpython/file/tip/Tools/msi/exe/crtlicense.txt
+.. |redist| replace:: List of redistributables
+.. _redist: https://msdn.microsoft.com/en-us/library/8kche8ah(v=vs.90).aspx
+
+Python compiled with Visual Studio (official) has some files from Microsoft
+and you are required to protect them if you distribute them. You can do that
+by including names of the files and a reworded version of |py2crt|_ or
+|py3crt|_ depending which interpreter you use, so that it targets the end-user
+of your application.
+
+* |redist|_
+
+Other libraries
+~~~~~~~~~~~~~~~
+
+* `zlib <https://github.com/madler/zlib/blob/master/README>`_
+
+.. note:: Please add other libs that you *don't use directly* and are present
+   after packaging with e.g. PyInstaller on Windows.
+
+Linux
+-----
+
+Missing.
+
+Mac
+---
+
+Missing.
+
+iOS
+---
+
+Missing.
+
+Android
+-------
+
+As APK is just an archive of files, you can extract files from it and (as in
+Windows part) check all the files.
+
+``APK/assets/private.mp3/private.mp3/`` contains all the included files. Most
+of them are related to Kivy, Python or your source, but those that aren't need
+checking.
+
+Known packages:
+
+* `pygame <https://bitbucket.org/pygame/pygame/src/a9c9f5bf17445dfc8f7a85b9c5222dbcb3ece3bb/LGPL>`_
+  (if old_toolchain is used)
+* `sqlite3 <https://github.com/ghaering/pysqlite/blob/master/LICENSE>`_
+* `six <https://bitbucket.org/gutworth/six/src/ca4580a5a648fc75abc568907e81abc80b05d58c/LICENSE>`_
+
+There are included libraries either Kivy directly or through Pygame/SDL2 uses,
+those are located in ``APK/lib/armeabi/``. Most of them are related to
+dependencies or are produced from python-for-android and are part of its source
+(and licensing).
+
+* libapplication.so
+
+RPi
+---
+
+Missing.
+
+Avoiding binaries
+-----------------
+
+.. |cons| replace:: consequences
+.. _cons: http://programmers.stackexchange.com/a/234295
+
+There is a way how to avoid this licensing process with avoiding creating
```

Comment:
This should also be stated, rather than authoritative, it should be more suggestive.